uftrace.c

```diff
@@ -994,6 +1014,7 @@ int main(int argc, char *argv[])
 	outfp = stdout;
 
 	argp_parse(&argp, argc, argv, ARGP_IN_ORDER, NULL, &opts);
+	check_shmem_freespace();
```

Comment:
You need to move it to `command_record()` as replay path doesn't need a shared memory.